src/index.ts

```diff
@@ -0,0 +1,28 @@
+import * as _RadonRetrievals from "./lib/radon/retrievals"
+export const RadonRetrievals: typeof _RadonRetrievals = _RadonRetrievals;
```

Comment:
We don't need to import it using an underscore because `RadonRetrievals` is a class with the `RadonRetrievals` type which makes it unnecessary. 

```suggestion
// if we aren't using RadonRetrievals in the file so we only want to export it, we can avoid the import line
export * as RadonRetrievals from "./lib/radon/retrievals";
```